x-pack/filebeat/input/netflow/decoder/v9/lru.go

```diff
@@ -53,6 +53,19 @@ type pendingTemplatesCache struct {
 	events  map[SessionKey][]*bytes.Buffer
 }
 
+const (
+	maxPendingPerKey = 256
+	maxTotalPending  = 8192
```

Comment:
How did we select those numbers? Do we know that the queues don't have more in the regular operation?